Sandbox/src/Game/Commands/DrinkCommand.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "Game/Commands/ICommand.h"
+#include "Game/Player/Player.h"
+#include "Game/Items/ItemFactory.h"
+
+class DrinkCommand : public ICommand
+{
+private:
+	Player* mPlayer;
+
+public:
+	DrinkCommand(Player* player)
+	{
+		mPlayer = player;
+	}
+
+	~DrinkCommand() = default;
+
+	bool CanExecute()
+	{
+		int activeSlot = mPlayer->Inventory().ActiveSlot();
+
+		if (Potion* food = dynamic_cast<Potion*>(mPlayer->Inventory().GetItem(activeSlot)))
```

Comment:
should "food" be "drink here? tsk tsk you plagiarizer

Reply:
Good catch lol, typo